### ISSUE TITLE
Add the focus style for the Switch component

### ISSUE THREAD

**Issue by tkeiyama** [tkeiyama/react-ui#75]
## Why

Every component needs the focus style for a11y. But the Switch component doesn't have it.
So we need to implement it.

## How

ref.
- https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active
- https://flowbite.com/docs/forms/toggle/#toggle-example
